.travis.yml

```diff
@@ -35,16 +61,30 @@ jobs:
       # Docs and lint python versions must be synced with those in tox.ini
     - python: '3.8'
       env: TOX_ENV=docs
+    - python: '3.8'
+      arch: arm64
+      env: TOX_ENV=docs
     - python: '3.8'
       env: TOX_ENV=lint
   allow_failures:
     # Currently failing due to https://foss.heptapod.net/pypy/cffi/issues/458
     - python: nightly
 
 install:
+    - |
+      if [[ -n ${BUILD_WHEEL} ]]; then
+         # To test it's own built wheel
+         export CIBW_TEST_COMMAND="cd {project} && tox -e $TOX_ENV"
```

Comment:
Are you sure this is testing the built wheel? I thought `tox` would do its own build unless you pass `--installpkg` and tell it where the built wheel lives

Reply:
I'm quite sure that it was testing the built wheel only, Here is the reference:
[cibuildwheel](https://cibuildwheel.readthedocs.io/en/stable/options/#test-command)
But anyway, Modified the same as you suggested, Now testing the built wheel using `--installpkg` with `tox` command.

Reply:
That link doesn't really answer the question. Installing the built wheel in the top-level environment doesn't install it in the environments tox creates for the tests themselves. I'm pretty sure `--installpkg` is necessary here.